docs/addon.md

```diff
@@ -13,14 +13,34 @@ myst:
 
 (chapter-addons)=
 
-# Add-on
+# Add-ons
+
+This chapter describes how to get, update, install, uninstall, and manage Plone add-ons.
+
 
 (addons-get-addons)=
 
 ## Get add-ons
 
 To get all the add-ons present in the current Plone site, use the {func}`api.addon.get_addons` function.
 The function accepts an optional `limit` parameter to filter the returned add-ons.
+`limit` may be one of the following strings.
+
+`available`
+:   Products that are not installed, but could be.
+
+`broken`
+:   Uninstallable products with broken dependencies.
+
+`installed`
+:   Only products that are installed and not hidden.
+
+`non_installable`
+:   Non-installable products.
+
+`upgradable`
+:   Only products with upgrades.
+The following examples demonstrate usage of the {func}`api.addon.get_addons` function.
```

Comment:
Whoopsie! This needs a blank line, else it becomes a definition list parsing error.

```suggestion

The following examples demonstrate usage of the {func}`api.addon.get_addons` function.
```